cg/services/order_validation_service/workflows/balsamic/utils.py

```diff
@@ -7,6 +7,7 @@
 def is_sample_missing_capture_kit(sample: BalsamicSample, store: Store) -> bool:
     application: Application = store.get_application_by_tag(sample.application)
     return (
-        application.prep_category == SeqLibraryPrepCategory.TARGETED_GENOME_SEQUENCING
+        application
```

Comment:
this fixes a bg in the validation, if the application does not exists, the validation breaks